karma.conf.js

```diff
@@ -7,8 +7,19 @@ function depMain(name) {
   return './node_modules/' + name + '/' + main;
 }
 
+//  getDeps :: String -> StrMap String String
+function getDeps(path) {
+  return Object.keys(require(path).dependencies);
+}
+
 //  dependencies :: Array String
-var dependencies = Object.keys(require('./package.json').dependencies);
+var dependencies = getDeps('./package.json')
+      .map(function(x) {
+        return [x, getDeps('./node_modules/' + x + '/package.json')];
+      })
```

Comment:
Doesn't this need to be recursive? What about the dependencies of our dependencies' dependencies? :stuck_out_tongue_winking_eye: